src/pages/studyView/charts/ChartContainer.tsx

```diff
@@ -143,7 +149,7 @@ export class ChartContainer extends React.Component<IChartContainerProps, {}> {
             resetFilters: action(() => {
                 this.props.onResetSelection(this.props.chartMeta, []);
             }),
-            onValueSelection: action((values: string[]) => {
+            onValueSelection: action((values: any) => {
```

Comment:
necessary?

Reply:
As I remember, I changed this to `any` because it was effectively `any` due to the way `onValueSelection` is used. Because `handlers` is of type `any`, any function within it is also loosely typed. This was already being abused by the `ChartTypeEnum.MUTATED_GENES_TABLE` case, which has `onUserSelection={this.handlers.onValueSelection}` despite its `onUserSelection` being of type `(value: string[][]) => void`

Reply:
got it, makes sense.